resources/views/admin.blade.php

```diff
@@ -169,6 +173,7 @@
                     $('.form-modal').attr("id","form-create");
                     $('#form-title').text("New Employee Details");
                     $('#btn-submit').text("Add Employee");
+                    $('.trash').addClass("d-none");
```

Comment:
have you tried to dynamically toggle the visibility ?

Reply:
Yes idk why some jquery methods are not working... @Raza9798 #56

Reply:
instead of adding and removing use `toggleClass` attribute

Reply:
sure will do...